citi-onboarding/Agreste-Consultoria-Jr
# Feat/keystone model 1

## What I did

- Models of the backend (Banner, Conteudos and QuemSomos);
- Routes;

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. git checkout develop
   1. git pull
   1. git checkout Feat/Keystone-model_1
   1. git pull origin Feat/Keystone-model_1
   1. npm install
   1. code .
   1. npm start

1. In your browser go to http://localhost:3001/admin

1. Check with the models appear in the keystone interface 

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description
